apps/src/aiComponentLibrary/warningModal/warning-modal.module.scss

```diff
@@ -20,6 +24,17 @@
   }
 }
 
+.headerContainer {
+  display: flex;
+  height: 40px;
```

Comment:
nit: I wonder if it's better to use `rem` value here, since Heading3 uses rems for font sizes, meaning px size can be different from 40px when base font size is changed